### ISSUE TITLE
obslog file is written to wrong location

### ISSUE THREAD

**Issue by MNBrod** [pypeit/PypeIt#1461]
Setup is writing the `.obslog` file to where setup was invoked from, not in the output directory. In `pypeitsetup.py` we see `_output_path` correctly joined to `sorted_file`, but that step is missing for `obslog`:

```

        if setup_only:
            # Collate all matching files and write .sorted Table (on pypeit_setup only)
            #if sort_dir is not None:
            #    sorted_file = os.path.join(sort_dir, os.path.split(sorted_file)[1])
            sorted_file = pypeit_file.replace('.pypeit', '.sorted')
            sorted_file = os.path.join(_output_path, os.path.split(sorted_file)[1])
            if not no_write_sorted:
                self.fitstbl.write_sorted(sorted_file, write_bkg_pairs=write_bkg_pairs,
                                      write_manual=write_manual)
            msgs.info("Wrote sorted file data to {:s}".format(sorted_file))
...
        if obslog:
            log_file = pypeit_file.replace('.pypeit', '.obslog')
            header = ['Auto-generated PypeIt Observing Log',
                      '{0}'.format(time.strftime("%a %d %b %Y %H:%M:%S", time.localtime()))]
            self.fitstbl.write(output=log_file, columns='pypeit', sort_col='mjd',
                               overwrite=True, header=header)
```

**Comment by kbwestfall** [comment]
I sneaked this fix into #1488 .

**Comment by kbwestfall** [comment]
Addressed and merged.
